packages/docs-app/package.json

```diff
@@ -57,5 +57,5 @@
     "docs"
   ],
   "author": "Palantir Technologies",
-  "license": "Apache-2.0"
+  "license": "UNLICENSED"
```

Comment:
Even if these packages are not published to npm, we still need to provide a license for when people check out the code from github. So, just add the LICENSE file to these packages and change to "SEE LICENSE FILE"